cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -139,20 +139,6 @@ enum class SurfaceTransform : uint32_t {
 CC_ENUM_CONVERSION_OPERATOR(SurfaceTransform);
 
 enum class Feature : uint32_t {
-    COLOR_FLOAT,
-    COLOR_HALF_FLOAT,
-    TEXTURE_FLOAT,
-    TEXTURE_HALF_FLOAT,
-    TEXTURE_FLOAT_LINEAR,
-    TEXTURE_HALF_FLOAT_LINEAR,
-    FORMAT_R11G11B10F,
-    FORMAT_SRGB,
-    FORMAT_ETC1,
-    FORMAT_ETC2,
-    FORMAT_DXT,
-    FORMAT_PVRTC,
-    FORMAT_ASTC,
-    FORMAT_RGB8,
```

Comment:
will it break compatibility?

Reply:
All references have been updated, but yes, this is a minor breaking change